pilot/pkg/features/pilot.go

```diff
@@ -31,14 +31,14 @@ import (
 
 var (
 	MaxConcurrentStreams = env.Register(
-		"ISTIO_GPRC_MAXSTREAMS",
```

Comment:
Not sure if any one is using this env vars. It will break customers who use these env var to configure these flags.